openslides_backend/presenter/check_mediafile_id.py

```diff
@@ -37,7 +48,99 @@ def get_result(self) -> Any:
 
         if not mediafile or mediafile.get("is_directory"):
             return {"ok": False}
-
-        # TODO: Call to the permission service.
+        self.check_permissions()
 
         return {"ok": True, "filename": mediafile["filename"]}
+
+    def check_permissions(self) -> None:
+        # superadmin is an admin of a meeting
+        if has_organization_management_level(
+            self.datastore, self.user_id, OrganizationManagementLevel.SUPERADMIN
+        ):
+            return
+        # The user is an admin of the meeting
+        mediafile = self.datastore.get(
+            FullQualifiedId(Mediafile.collection, self.data["mediafile_id"]),
+            [
+                "meeting_id",
+                "used_as_logo_$_in_meeting_id",
+                "used_as_font_$_in_meeting_id",
+                "projection_ids",
+                "is_public",
+                "inherited_access_group_ids",
+            ],
+        )
+        meeting_id = mediafile.get("meeting_id")
+        if not meeting_id:
+            raise PermissionDenied("You are not allowed to see this mediafile.")
+
+        meeting = self.datastore.get(
+            FullQualifiedId(Collection("meeting"), meeting_id),
+            ["admin_group_id", "enable_anonymous", "user_ids", "committee_id"],
+        )
+        groups_field = f"group_${meeting_id}_ids"
+        user = self.datastore.get(
+            FullQualifiedId(Collection("user"), self.user_id), [groups_field]
+        )
+        if meeting.get("admin_group_id") in user.get(groups_field, []):
+            return
```

Comment:
Please move this to a helper function `is_admin` (together with the above check for superadmin) and use that here and [in this meeting.update test](https://github.com/OpenSlides/openslides-backend/blob/6db4f187b50e07900569a287b4ae541a0f6cf12c/openslides_backend/action/actions/meeting/update.py#L228) (and in any other places if you find them, on a quick search I did not find more)

Reply:
Moved a is_admin function into the permission_helpers and use it at meeting.update.